.tests/integration/config.yaml

```diff
@@ -2,20 +2,24 @@ samples: "samples.tsv"
 units: "units.tsv"
 resources: "resources.yaml"
 
-default_container: "docker://hydragenetics/common:0.1.1"
+default_container: "docker://hydragenetics/common:0.1.3"
 
 reference:
   fasta: "reference/WASH7P.fna"
   dict: "reference/WASH7P.dict"
```

Comment:
Are `dict`and `fai` required for alignment? They are not used in any rule, are they?

Reply:
maybe we can skip them for this module then? What do you think @jonca79 @Smeds ?

Reply:
It is used by samtools_merge_bam

Reply:
I have removed .dict

Reply:
ok, good that you checked 👍